tracer/src/Datadog.Trace/Ci/Tags/CommonTags.cs

```diff
@@ -163,26 +163,26 @@ internal static class CommonTags
         /// <summary>
         /// Test Session Intelligent Test Runner tests skipping is enabled flag
         /// </summary>
-        public const string TestSessionTestsSkippingEnabled = "test_session.itr.tests_skipping.enabled";
+        public const string TestSessionTestsSkippingEnabled = "test.itr.tests_skipping.enabled";
 
         /// <summary>
         /// Test Session Code Coverage is enabled flag
         /// </summary>
-        public const string TestSessionCodeCoverageEnabled = "test_session.code_coverage.enabled";
+        public const string TestSessionCodeCoverageEnabled = "test.code_coverage.enabled";
 
         /// <summary>
         /// Test Module Intelligent Test Runner tests skipping is enabled flag
         /// </summary>
-        public const string TestModuleTestsSkippingEnabled = "test_module.itr.tests_skipping.enabled";
+        public const string TestModuleTestsSkippingEnabled = "test.itr.tests_skipping.enabled";
```

Comment:
This is identical to `TestSessionTestsSkippingEnabled`, and these aren't public, so can we remove it (and rename the other one)?

Reply:
true, changed here: [6ce8a02](https://github.com/DataDog/dd-trace-dotnet/pull/3909/commits/6ce8a021c82f272c0e42114babeb30e0109fcdb1)